client/my-sites/site-settings/publishing-tools/hooks/use-get-post-by-email.tsx

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+import type {
+	PostByEmail,
+	PostByEmailResponse,
+} from 'calypso/my-sites/site-settings/publishing-tools/types';
+
+export const getPostByEmailPath = ( siteId: number | null ) => `/sites/${ siteId }/post-by-email`;
+
+export const getCachePostByEmailKey = ( siteId: number | null ) => [
```

Comment:
```suggestion
export const getPostByEmailKeyQueryKey = ( siteId: number | null ) => [
```
I overlooked this in #90489, but I think we should mention "query key" rather than "cache key".